lms/djangoapps/instructor/tests/test_api.py

```diff
@@ -3293,6 +3293,7 @@ def test_get_anon_ids(self):
         self.assertTrue(
             body.endswith('"{user_id}","41","42"\n'.format(user_id=self.students[-1].id))
         )
+        self.assertIn("attachment; filename=org", response['Content-Disposition'])
```

Comment:
previously it was coming
`"attachment; filename=b'org.0-course_0-Run_0-anon-ids.csv'"`